configuration_broker/config_broker/config_broker.h

```diff
@@ -0,0 +1,145 @@
+// Copyright Configured Things and CHERIoT Contributors.
+// SPDX-License-Identifier: MIT
+
+#include "cdefs.h"
+#include "compartment-macros.h"
+#include "token.h"
+#include <atomic>
+#include <compartment.h>
+#include <locks.hh>
+
+//
+// Internal representation of a token which allows an operation on
+// a configuration item.  This is put in a Capability sealed with one
+// of three different keys to define how it can be used.
+// Size and update_interval only used when setting the parser, but
+// keeping a common struct keeps the code a bit cleaner.
+//
+struct ConfigToken
+{
+	uint16_t   id;             // id for the capability, assigned on first use
+	size_t     size;           // size of the item
+	uint32_t   updateInterval; // Min MS between updates
```

Comment:
Here, where the capability is being defined, its mS.     In the internal record of the broker its stored as ticks (in minTicks) to avoid converting it each time.   Can you point me to where the docs are inconsistent so I can make it clearer ?